VimeoNetworkingExample-iOS/VIMUser/VIMUserTests.swift

```diff
@@ -83,6 +83,10 @@ class VIMUserTests: XCTestCase
                     XCTAssertEqual(analyticsIdentifier, "live_business")
                 case .livePremium:
                     XCTAssertEqual(analyticsIdentifier, "live_premium")
+                case .proUnlimited:
```

Comment:
I think we need to add more to the unit tests to cover these two cases.  Codecov is reporting that our test coverage drops with this PR.